app/forms/otp_verification_form.rb

```diff
@@ -13,6 +13,8 @@ def submit
   attr_reader :code, :user
 
   def valid_direct_otp_code?
+    code_length = Devise.direct_otp_length
+    return false unless code =~ /^\d{#{code_length}}$/
```

Comment:
Yes. That's what I meant to do but forgot to push. I thought there was a Rubocop setting that detects that, but it looks like it only looks for it when you use the Rails format validation.